tools/loris_db_upgrade.php

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/php
+<?php
+
+/**
+ * This script will upgrade the loris database to the lastest. 
+ * Usage: php loris_db_upgrade.php [-D]
+ *
+ * PHP Version 7
+ *
+ * @category Install
+ * @package  Loris
+ * @author   Shen Wang <shen.wang2@mcgill.ca>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ * @package behavioural
+ */
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once __DIR__ . "/generic_includes.php";
+ set_include_path(get_include_path().":../project/libraries:../php/libraries:");
+require_once "../php/libraries/Database.class.inc";
+require_once "../php/libraries/NDB_Config.class.inc";
+$path    = __DIR__ . "/../SQL/Release_patches";
+
+$files = array_diff(scandir($path), array('.', '..'));
+
+echo "This script will upgrade the loris database to the lastest.\n";
+echo "Backing Up the Database Before Upgrading the Database.\n"
+
+$v = readline("\n Enter current version of Loris database (ex:21.0): ");
+  
+if (!file_exists('../project/config.xml')) {
+    die("config.xml file doesn't exist. Aborting update.");
+}
+
+$DB = \Database::singleton();
+$sql = null;
+foreach($files as $file){
+
+    if (substr($file, 0, 5)>=$v) {
+     print_r($file);
+      $a = file_get_contents(__DIR__ . "/../SQL/Release_patches/".$file);
+      $sql .= $a;
+
+    }
+}
+    try {
+       $DB->_PDO->exec($sql);
```

Comment:
schema upgrade will fail as any modification to the schema (ALTER TABLE, DROP TABLE, CREATE TABLE, ...) require the `quatuser`.
Only the `INSERT, UPDATE, DELETE` modification would succeed. 
@johnsaigle created some PR to facilitate the upgrade process, maybe you could improve these.

Reply:
I have a tool to update the Raisinbread database but it's not safe to use for upgrading production. It deletes all data. As a result I don't think it's useful in this context.

Reply:
@johnsaigle I though that you had started work to update a production environment.

Reply:
I don't think so. Most of the work I've done recently is in streamlining developer instances of LORIS. I'd be happy if something like this PR goes ahead though.